js/webp-fix.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * webp-fix.js
+ *
+ *	Detects lack of webp support, loads full images instead. 
+ *
+ * Usage:
+ *   $config['additional_javascript'][] = 'js/jquery.min.js';
+ *   $config['additional_javascript'][] = 'js/webp-fix.js';
+ *
+ */
+
+$(document).ready(function() {
+
+	// from: https://stackoverflow.com/a/54120785
+	function testWebP() {
+		return new Promise(res => {
+			const webP = new Image();
+			webP.src = 'data:image/webp;base64,UklGRjoAAABXRUJQVlA4IC4AAACyAgCdASoCAAIALmk0mk0iIiIiIgBoSygABc6WWgAA/veff/0PP8bA//LwYAAA';
+			webP.onload = webP.onerror = () => {
+				res(webP.height === 2);
+			};        
+		})
+	};
+
+	testWebP().then(hasWebP => {
+		if(!hasWebP){
+			$('div.file a').each(function(index){
+				let ext = ["png","jpg", "jpeg"];
+				let file = $(this);
+				let fullSize = file.prop("href");
+				if (ext.includes(fullSize.split('.').pop())){
+					file.find('img').prop("src", fullSize);
+			  	}
```

Comment:
Is this just loading full sized images?

This isn't a work-around! This will cause a massive amount of content to get loaded, and high res images will have to be rendered out by some poor old browser. From experience in trying to display a directory of images on a page without making thumbs - this is a very bad idea.

Reply:
@towards-a-new-leftypol 
> Some alternatives to loading every image in native resolution:
> 
>     * use js to decode webp:
>       https://github.com/webmproject/libwebp-demo/tree/gh-pages/webp_js
> 
>     * just revert back to jpg thumbnails and use a better encoder
> 
>     * create two thumbs for each file and serve the legacy format as a fallback.
> 

Before I fix anything else, we need to establish whether we want to fix this.
>  * use js to decode webp:
This is extremely slow. It is quicker to load the full resolution images. As long as the images are not enormous of course. Especially on a cellphone, decoding with js is a no go.
> * just revert back to jpg thumbnails and use a better encoder
We need to decide whether we want this. I'm not opposed. 
> * create two thumbs for each file and serve the legacy format as a fallback.
Tbh, I'm not willing to put in the work. It requires adding fields to the backend and adding JS too, since the `<picture>` tag, as recommended by google, is even less supported. So this wouldn't be a "native" solution either. 

This is for a small minority of users. Right now this is broken. Some of our users cannot see ANY thumbnails on the board. Yeah, a "fix" is definitely a misnomer, but this is a sev1 bug. Unless there's a better fix on the way, we need to mitigate this ASAP. 

Roping in @discomrade for this discussion. I don't care what we do, but we need to decide what to do quickly. Otherwise the sev1 issue will persist.